src/components/flat-table/flat-table-row/flat-table-row.component.tsx

```diff
@@ -293,6 +299,7 @@ export const FlatTableRow = React.forwardRef<
         id={internalId.current}
         data-selected={selected && expandableArea === "wholeRow"}
         data-highlighted={highlighted && expandableArea === "wholeRow"}
+        rowHeight={rowHeight}
```

Comment:
```suggestion
        rowHeight={rowRef?.current?.offsetHeight}
```

**suggestion** (non-blocking): as talked about in comment above, this is an alternative to using state